src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/OuterTypeNumberCheckTest.java

```diff
@@ -33,10 +33,16 @@
 import com.puppycrawl.tools.checkstyle.utils.CommonUtils;
 
 public class OuterTypeNumberCheckTest extends BaseCheckTestSupport {
+
+    /**
+     * Prefix for input directory of test case
+     */
+    private static final String TEST_RESOURCE_PATH_PREFIX = "checks" + File.separator
+        + "sizes" + File.separator + "outertypenumber" + File.separator;
+
```

Comment:
please move it back to method, all other tests have it there, so lets keep the same format.